proto/messages.proto

```diff
@@ -192,6 +192,7 @@ message Asset {
     optional string asset_id = 3;
```

Comment:
Actually, old clients that only understand AES-CBC, if we send them AES-GCM, won't be able to decrypt the asset anyway. So does it make sense to maintain "backward compatibility" here?